content/en/docs/setup/install/multicluster/single-network/remote/index.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Add a Remote Cluster
+description: Describes how to add a remote cluster to your mesh on a flat network.
+weight: 30
+keywords: [multicluster, remote-cluster, deployment]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, you can add a
+{{< gloss >}}remote cluster{{< /gloss >}} to your
+{{< gloss >}}service mesh{{< /gloss >}}.
+
+To give you clean instructions, this section starts right after you complete the
+initial configuration, and assumes that `Cluster_1` and `Cluster_2` reside on
+the same network. The new remote cluster, `Cluster_2` in this section, shares
+the {{< gloss >}}control plane{{< /gloss >}} of `CLUSTER_1`, and this guide
+assumes that `Cluster_1` and `Cluster_2` reside on the same network.
```

Comment:
I know this isn't a tutorial, but as it reads like one, why don't you call them CLUSTER_1, CLUSTER_2, CLUSTER_3 (or even better CLUSTER_PRIMARY, CLUSTER_SECONDARY, CLUSTER_REMOTE or such) and then you can run either of these two and have it all work?

Reply:
`CLUSTER_SECONDARY` wouldn't be a good name, since we have no concept of "secondary" in Istio. It could be `PRIMARY1`, `PRIMARY2`, and `REMOTE`, however.

Reply:
The naming schema is due to the content reusability and modularity requirements. Using the names you propose, would mean no longer being able to reuse content across docs causing an increase in maintenance overhead.